theme/views/components/shape.latte

```diff
@@ -0,0 +1,7 @@
+{declare string $name}
+{declare string $assetsPath}
+{declare string $buildstamp = getBuildstamp() ?? ''} {* @TODO: find out why buildstamp is not available without calling getBuildstamp() *}
```

Comment:
`{component ...}` components are sandboxed, so they get just a few "global" variables, I guess $buildstamp is not one of those.

Reply:
Fixed: https://github.com/manGoweb/MangoPress/pull/203/commits/e4f473b7f8c8b37d376128a836096f599ff6ee0e